doozer/doozerlib/cli/scan_fips.py

```diff
@@ -72,7 +83,53 @@ async def am_i_root(self):
         self.runtime.logger.info(f"Running as user {out}")
         return rc == 0 and "root" in out
 
+    def get_tagged_latest(self, tag):
+        """
+        Returns the latest RPMs and builds, triggered by the automation, tagged in to the brew tag received as input
+        """
+        latest_tagged = self.koji_session.listTagged(tag=tag, latest=True, owner="exd-ocp-buildvm-bot-prod")
+        if latest_tagged:
+            return latest_tagged
+        else:
+            return []
+
+    def get_all_latest_builds(self):
+        brew_tags = []
+        # Setup brew tags
+        tags = self.runtime.get_errata_config()["brew_tag_product_version_mapping"].keys()
```

Comment:
you can use
`brew_tags = self.runtime.gitdata.load_data(key='erratatool', replace_vars=runtime.group_config.vars).data.get('brew_tag_product_version_mapping', {}).keys()`

Reply:
Can I know what the difference is? Is the value is already in `get_errata_config` then we can use it directly right?

Reply:
Nice, thanks for pointing that out!